code/game/objects/items/implants/implant_stealth.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/implant/stealth
+	name = "S3 implant"
+	desc = "Allows you to be hidden in plain sight."
+	actions_types = list(/datum/action/item_action/agent_box)
+
+//Box Object
+
+/obj/structure/closet/cardboard/agent
+	name = "inconspicious box"
+	desc = "It's so normal that you didn't notice it before."
+	icon_state = "agentbox"
+	move_speed_multiplier = 0.5
+
+/obj/structure/closet/cardboard/agent/Initialize()
+	. = ..()
+	START_PROCESSING(SSfastprocess, src)
```

Comment:
why are you using a subsystem ticker? animate the alpha when it updates with a timer if necessary, you're ticking for no reason most of the time.